public/js/get_events.js

```diff
@@ -0,0 +1,22 @@
+/* global XMLHttpRequest */
+var module = (function () {
+  console.log(module);
+  return {
+    getEvents: function (path, callback) {
+      var baseUrl = 'https://nazareth-open-tourism-platform.herokuapp.com/events';
+      var url = baseUrl + path;
+      var xhr = new XMLHttpRequest();
+      xhr.onreadystatechange = function () {
+        if (xhr.readyState === 4 && xhr.status === 200) {
+          console.log('date from', xhr.responseText);
+          callback(null, JSON.parse(xhr.responseText));
+        } else if (xhr.status === 500) {
+          var errorMessage = new Error('Server Error! Status: ' + xhr.status);
+          callback(errorMessage, null);
+        }
```

Comment:
what if the status is neither 200 nor 500?

Reply:
400 page not found

Reply:
but the function does not test for this.